contactmoment/src/main/kotlin/com/ritense/contactmoment/web/rest/request/CreateContactMomentRequest.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.contactmoment.web.rest.request
+
+data class CreateContactMomentRequest(
+    val kanaal: String,
```

Comment:
I think the kanaal wasn't part of the story. If we want to do this i think it should be an enum. Something like: Phone, Mail, Other

Reply:
I removed kanaal from the request for now. we can always add it later.
And made kanaal enum